tests/testthat/testImportSNV.TCGA.R

```diff
@@ -0,0 +1,208 @@
+# testImportSNV.TCGA.R
```

Comment:
Generally: turn writeLog off in all tests except in one block where you specifically test log output. Make sure there is no log file before you test, and after your tests delete the one you created. Log files are being appended to, and if you don't take care, your tests will fail because of old contents.